### PR TITLE
style: fix misspelled 'parse'; and minor improvements on docs

### PR DESCRIPTION
**Pull request by micalevisk**
I've read the whole docs and template files. Didn't find any other typo.

### CODE REVIEW

File: apps/docs/docs/schematics/usage.md [hunk 1]
```diff
@@ -5,30 +5,52 @@ sidebar_label: Usage
...
+--sourceRoot=<name>   NestJS service source root directory.
+--flat                Whether or not a directory is created. (default: fale)
+--spec                Whether or not a spec file is generated. (default: true)
+--default             Whether or not the command is the default command for the CLI. (default: false)
+--question=<name>     The name of the related question set.
+```
 
 ### Generating Commands with Questions
 
 As mentioned above, you can use the `command` schematic and provide a question set name. You can also use `--question=<name>` to provide a name for the question set without waiting to answer the prompt.
```
Comment (micalevisk) [inline apps/docs/docs/schematics/usage.md:36]: do notice that we can't pass any option to `command` schematics using `@nestjs/cli`

```
$ nest g -c nest-commander-schematics command --question=name
error: unknown option '--question'

$ nest g -c nest-commander-schematics --question=name command
error: unknown option '--question'
```

Comment (jmcdo29) [reply apps/docs/docs/schematics/usage.md:36]: Oh really? I swear I had that tested. Maybe it was with the ng CLI instead. I'll need to see what can be done about this

Comment (micalevisk) [reply apps/docs/docs/schematics/usage.md:36]: do you think Nest CLI should support both usages (options along with `-c`, and the usual one when using custom collection)?

Comment (micalevisk) [reply apps/docs/docs/schematics/usage.md:36]: looks like the `--name` option is being populated with the first arg

```
$  nest g command foo
? What is the name of the related question set? (Leave blank for none) 
Failed to execute command: node nest-commander-schematics:command --name=foo --no-dry-run --language="ts" --sourceRoot="src" --no-spec
```

**Comment by micalevisk** [comment]
image

image

I believe the above should be `false` as Inquirer(?) uses `'input'` by default

File: apps/docs/docs/schematics/usage.md (hunk 1 above)
Comment (jmcdo29) [reply apps/docs/docs/schematics/usage.md:36]: I think I'll need to look into fixing this in nestjs/nest-cli. Thanks for bringing it up

**Comment by jmcdo29** [comment]
> image
> 
> image
> 
> I believe the above should be `false` as Inquirer(?) uses `'input'` by default

Good catch. Let's document that as `false`

**Review by jmcdo29 — APPROVED** [review]

**Pull request merged**